assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -59,6 +60,7 @@ default_irq_smp_affinity = ignore
 {{end}}
 
 [sysctl]
+uname_regex=rt
```

Comment:
You would have to split the section here, not all the options are RT kernel specific. Moreover, this will not work, because we inherit from other profiles that have the same sysctls. This needs to be fixed on tuned level.